ui/v2.5/src/components/MainNavbar.tsx

```diff
@@ -14,6 +14,8 @@ import { SessionUtils } from "src/utils";
 import { Icon } from "src/components/Shared";
 import { Manual } from "./Help/Manual";
 
+const Mousetrap = require("mousetrap-pause")(require("mousetrap"));
+
```

Comment:
It's not really necessary to use require . You can import the modules normally:
```
import Mousetrap from 'mousetrap';
import MousetrapPause from 'mousetrap-pause';
MousetrapPause(Mousetrap); 
```

You'll also have to add `declare module "mousetrap-pause";` to `src/globals.d.ts`.